### DISCUSSION TITLE
增强自定义顶栏“收藏”的搜索功能

### DISCUSSION THREAD

**Discussion by MC-dusk** [the1812/Bilibili-Evolved#2742] [功能建议 / Ideas]
现在的情况是搜索后，按稿件标题关键词匹配，区分大小写，而且有时会有奇怪的bug搜不出来，比如关键词只有一个字，标题有空格分开了关键词。和b站的收藏夹搜索效果是一样的。
希望加入不区分大小写、忽略空格等不严格匹配的搜索功能，并支持搜索up用户名（匹配方式同前）。
如果选定了某个收藏夹的话，就在这个收藏夹里面搜索，如果是默认收藏夹则搜索全部。

**Comment by Geeyun-JY3** [comment]
相关：#915

**Reply by MC-dusk** [reply]
啊，我没搜到，看来是远古巨坑了😅

**Reply by Geeyun-JY3** [reply]
拿出来晒晒也不是坏事（

**Comment by the1812** [comment]
那只能从当前加载出来的卡片里搜, 其他都是调用 b 站 API 搜的

**Reply by MC-dusk** [reply]
其实其他的随缘，但搜索up名字确实需求，我以前好像搜出来过，后来又不行了
